src/webgl/3d_primitives.js

```diff
@@ -1588,13 +1588,62 @@ p5.RendererGL.prototype.bezierVertex = function(...args) {
 
     const LUTLength = this._lookUpTableBezier.length;
 
+    // Get the last fill color
+    const _vertexColors = this.immediateMode.geometry.vertexColors;
+    const _vertexColorSize = _vertexColors.length;
+    const lastFillColor = [
```

Comment:
This might be more concise if we do `this.immediateMode.geometry.vertexColors.slice(-4)` to get the last 4 elements. Do we need to handle this array being empty? I think immediate mode always copies the fill color into this array, so probably not, right?

Reply:
That's more concise...I'd like to fix it! thank you.